pkgs/top-level/all-packages.nix

```diff
@@ -27576,6 +27576,10 @@ with pkgs;
     buildGoModule = buildGo121Module;
   };
 
+  tailscale-nginx-auth = callPackage ../by-name/ta/tailscale-nginx-auth/package.nix {
```

Comment:
TIL by-name! That's new :) It looks right though

Reply:
It's been introduced in https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md and active for new packages, mass-migration of existing packages will be done soon-ish. There's been a talk at NixCon about this https://media.ccc.de/v/nixcon-2023-35713-not-all-packages-anymore-nix